README.md

```diff
@@ -1,4 +1,5 @@
 # python-domino
+[![PyPI version](https://badge.fury.io/py/dominodatalab.svg)](https://badge.fury.io/py/dominodatalab)
```

Comment:
1. Why can't you just use https://pypi.org/project/dominodatalab?
2. Why this link is needed? PyPI is a very standard thing so everyone should be able to find a package there knowing only its name. Note if you add the link, then the next release of dominodatalab in PyPI (e.g. 1.0.9) will also have the link on the project page in PyPI (as README.md is shared)